src/components/TakeQuizPage/index.jsx

```diff
@@ -190,58 +190,77 @@ const RadioOptions = () => {
     const setSelectedOption = (event) => {
         updateSelectedOptions(questionId, [event.target.value]);
     };
+    const clearSelection = (event) => {
+        updateSelectedOptions(questionId, [event.target.value]);
+    }
 
     return (
-        <Radio.Group
-            onChange={setSelectedOption}
-            value={
-                question.selectedOptions.length > 0
-                    ? question.selectedOptions[0]
-                    : ""
-            }
-        >
-            <div className="question-options">
-                {options &&
-                    options.map(({ _id, text }) => {
-                        return (
-                            <Radio key={_id} value={_id}>
-                                <div className="question-option-text">
-                                    {text}
-                                </div>
-                            </Radio>
-                        );
-                    })}
+        <div>
+            <Radio.Group
+                onChange={setSelectedOption}
+                value={
+                    question.selectedOptions.length > 0
+                        ? question.selectedOptions[0]
+                        : ""
+                }
+            >
+                <div className="question-options">
+                    {options &&
+                        options.map(({ _id, text }) => {
+                            return (
+                                <Radio key={_id} value={_id}>
+                                    <div className="question-option-text">
+                                        {text}
+                                    </div>
+                                </Radio>
+                            );
+                        })}
+                </div>
+            </Radio.Group>
+            <div>
+                <Button 
```

Comment:
You can move this button and its onClick to the Question component itself, so it does not have to be placed in both RadioOptions and CheckboxOptions